plugins/frontend/backstage-plugin-cloudsmith/src/components/CloudsmithStatsCard/Content.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { CloudsmithStatsCardContentProps } from './types';
+import { CloudsmithClient, RepoStats } from '../../api';
+import {
+  discoveryApiRef,
+  fetchApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import { ErrorPanel, Progress } from '@backstage/core-components';
+import {
+  Paper,
+  Table,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableBody,
+  Card,
+  CardHeader,
+  Avatar,
+  CardContent,
+} from '@material-ui/core';
+
+/**
+ * A component to render the stats about a cloudsmith repo
+ *
+ * @public
+ */
+export const Content = ({ owner, repo }: CloudsmithStatsCardContentProps) => {
+  const fetchApi = useApi(fetchApiRef);
+  const discoveryApi = useApi(discoveryApiRef);
+  const cloudsmithApi = new CloudsmithClient({ fetchApi, discoveryApi });
```

Comment:
Cannot we do something like this ? That we ask for the cloudsmithApi with the cloudsmithApiRef ? 
```
  import { CloudsmithClient, RepoStats, cloudsmithApiRef } from '../../api';
  ...
```
```suggestion
  const cloudsmithApi = useApi(cloudsmithApiRef);
```

Reply:
It doesnt seem to work for home plugins, I am not sure why.